db/init/00-database.sql

```diff
@@ -260,8 +305,8 @@ DECLARE
     batch base.batch;
 BEGIN
     -- Create pending batch
-    INSERT INTO base.batch (status, indicator_group_id)
-    VALUES ('Pending', indicator_group_id)
+    INSERT INTO base.batch (status, indicator_group_id, user_group)
+    VALUES ('Pending', indicator_group_id, '')
```

Comment:
Why is `user_group` here always an empty string?

Reply:
because we currently do not get the group information for indicators, batches etc. yet so this is a placeholder until then. We added a TODO comment to reflect this

Reply:
@Moeby also document this in a requirement.